docker/yb-voyager-docker

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+argv=( "$@" )
+
+i=0
+while [ $i -lt $# ]
+do
+    if [[ ${argv[${i}]} == "--export-dir" ]]
+    then
+        j=$(( $i + 1))
+        vol="-v ${argv[$j]}:/export-dir"
+        argv[$j]='/export-dir'
+    fi
+    i=$(( $i + 1))
+done
+
+dockerCmd="docker run ${vol} -it --network=host --rm --privileged localhost/yb-voyager:0.82 yb-voyager ${argv[*]}"
+
+echo $dockerCmd
```

Comment:
Comment out this line. It is okay for debugging. But we don't want to show it as part of normal output.